mappings/net/minecraft/item/Item.mapping

```diff
@@ -219,6 +229,7 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 			COMMENT
 			COMMENT @return this instance
 			ARG 1 recipeRemainder
+		METHOD m_pvohmnyq jukeboxPlayable (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;
```

Comment:
```suggestion
		METHOD m_pvohmnyq jukeboxSong (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;
			ARG 1 song
```
`C_yeazlrrn` is `JukeboxSong` in #607 (same as MojMap and Yarn) (so this takes a `RegistryKey<JukeboxSong>`)